apps/meteor/ee/server/apps/marketplace/getMarketplaceHeaders.ts

```diff
@@ -0,0 +1,9 @@
+import { Info } from '../../../../app/utils/rocketchat.info';
+
+const appsEngineVersionForMarketplace = Info.marketplaceApiVersion.replace(/-.*/g, '');
+
+export function getMarketplaceHeaders(): Record<string, any> {
```

Comment:
Do you need this one to be a func? you can export the object directly 👀 as it apparently won't change during the runtime.